native/mod_manager/mod_manager.c

```diff
@@ -267,18 +267,18 @@ static int loc_worker_nodes_are_updated(void *data, unsigned int last)
 static apr_status_t lock_memory(apr_file_t *file, apr_thread_mutex_t *mutex)
 {
     apr_status_t rv;
-    rv = apr_file_lock(file, APR_FLOCK_EXCLUSIVE);
+    rv = apr_thread_mutex_lock(mutex);
     if (rv != APR_SUCCESS)
         return rv;
-    rv = apr_thread_mutex_lock(mutex);
+    rv = apr_file_lock(file, APR_FLOCK_EXCLUSIVE);
     if (rv != APR_SUCCESS)
-        apr_file_unlock(file);
+        apr_thread_mutex_unlock(mutex);
     return rv;
 }
 static apr_status_t unlock_memory(apr_file_t *file, apr_thread_mutex_t *mutex)
 {
-    apr_thread_mutex_unlock(mutex);
-    return(apr_file_unlock(file));
+    apr_file_unlock(file);
+    return(apr_thread_mutex_unlock(mutex));
```

Comment:
space after return

Reply:
Is it necessary to make it into a separate PR? I think a commit with an interactive rebase (it can reorder and squash commits) should be fine.